Sming/Components/MultipartParser/MultipartParser.h

```diff
@@ -14,33 +14,44 @@
 
 #include <Network/Http/HttpCommon.h>
 #include <Network/Http/HttpRequest.h>
+#include <Data/Stream/ReadWriteStream.h>
 
 #include "multipart-parser/multipart_parser.h"
 
 class MultipartParser
 {
+	MultipartParser(HttpRequest& request, const String& boundary);
 public:
-	explicit MultipartParser(HttpRequest* request);
-	~MultipartParser();
+	static MultipartParser *create(HttpRequest& request);
```

Comment:
@aemseemann Please create a subdirectory `src` under MultipartParser. Move HttpMultipartResource.cpp, HttpMultipartResource.h, MultipartParser.cpp and MultipartParser.h to `src`. Add new empty `.cs` file in the `src` directory.  Modify the component.mk file to have
```
COMPONENT_SRCDIRS := src
COMPONENT_INCDIRS := src
```
After that from $SMING_HOME run `make cs`.
```